frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -0,0 +1,77 @@
+import * as loginView from '@console/internal-integration-tests/views/login.view';
+import { withResource, removeLeakableResource } from '@console/shared/src/test-utils/utils';
+import { createProject } from './utils/utils';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { VM_ACTION } from './utils/consts';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const {
+  BRIDGE_HTPASSWD_IDP = 'test',
+  BRIDGE_HTPASSWD_USERNAME = 'test',
+  BRIDGE_HTPASSWD_PASSWORD = 'test',
+} = process.env;
+const configName = ProvisionConfigName.URL;
+const leakedResources = new Set<string>();
+const provisionConfigs = getProvisionConfigs(testName);
+
+const provisionConfig = provisionConfigs.get(configName);
+provisionConfig.networkResources = [];
+provisionConfig.storageResources = [];
+
+describe('Kubevirt non-admin VM Creation', async () => {
+  beforeAll(async () => {
+    await browser.get(appHost);
+    await browser.sleep(3000); // Wait long enough for the login redirect to complete
+  });
+  describe('Non-Admin Flow Test', async () => {
+    beforeAll(() => {
+      // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    });
+    afterAll(() => {
+      // Set jasmine timeout interval back to the original value after these tests are done
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+    });
+
+    it('Non-Admin Create-VM and Delete Flow Test', async () => {
+      const testNonAdminNamespace = `${testName}-non-admin`;
+      await loginView.login(
+        BRIDGE_HTPASSWD_IDP,
+        BRIDGE_HTPASSWD_USERNAME,
+        BRIDGE_HTPASSWD_PASSWORD,
+      );
+      expect(browser.getCurrentUrl()).toContain(appHost);
+      expect(loginView.userDropdown.getText()).toContain(BRIDGE_HTPASSWD_USERNAME);
+
+      // Navigate to Virtual Machines page
+      await browser.get(`${appHost}/k8s/ns/${testNonAdminNamespace}/virtualmachines`);
+
+      // Check to make sure Access is Restricted.
+      await browser.wait(
+        until.textToBePresentInElement($('.cos-status-box__title'), 'Restricted Access'),
+      );
+
+      await createProject(testNonAdminNamespace);
+
+      const vm1Config = vmConfig(configName.toLowerCase(), provisionConfig, testNonAdminNamespace);
+      vm1Config.startOnCreation = false;
+
+      const vm = new VirtualMachine(
+        vmConfig(configName.toLowerCase(), provisionConfig, testNonAdminNamespace),
+      );
+      await withResource(leakedResources, vm.asResource(), async () => {
+        await vm.create(vm1Config);
+        await vm.action(VM_ACTION.Start);
+        await vm.action(VM_ACTION.Restart);
```

Comment:
I think we don't need to test all of the actions, I would only try to 
```
await vm.action(VM_ACTION.Start, false); // Without waiting for the VM to be Running
await vm.waitForStatus(VM_STATUS.Starting); // Just to make sure it is actually starting, 
```
and then we can delete it